### PR TITLE
WT-4755 switch to a better timestamp and transaction ID ordering convention

### PR DESCRIPTION
**Pull request by keithbostic**

### CODE REVIEW

File: src/include/cell.h [hunk 1]
```diff
@@ -154,12 +154,17 @@ struct __wt_cell_unpack {
 
 	uint64_t v;			/* RLE count or recno */
 
-	uint64_t start_txn, stop_txn;	/* Value validity window */
-	wt_timestamp_t start_ts, stop_ts;
+	wt_timestamp_t	start_ts;	/* Value validity window */
+	uint64_t	start_txn;
+	uint64_t	stop_txn;
+	wt_timestamp_t	stop_ts;
```
Comment (bvpvamsikrishna) [inline src/include/cell.h]: I am not able to understand why the `stop_txn` and `stop_ts` are interchanged here.

Comment (keithbostic) [reply src/include/cell.h]: Thanks, @bvpvamsikrishna. Fixed, pushed in 7d9942c.

**Comment by lukech** [comment]
margay-aws was unstable
retest this please

**Comment by keithbostic** [comment]
autobuild: retest this please

**Pull request merged**